install.sh

```diff
@@ -72,5 +72,5 @@ echo "# Emirge is now installed. Please run the following commands            #"
 echo "# to test the installation (assuming your shell is bash):               #"
 echo "# $ source $conda_prefix/bin/activate dgfem                             #"
 echo "# $ export OCL_ICD_VENDORS=$MY_CONDA_DIR/envs/dgfem/etc/OpenCL/vendors  #"
```

Comment:
Why is that needed? The ocl-icd (Linux)/khronos loader (MacOS) in Conda should default to this.

Reply:
Seems to be unnecessary, will remove.